participants/Martínez_Fernández/pr_tutorial/simple_functions.py

```diff
@@ -10,3 +10,14 @@ def factorial(value):
         return 1
     else:
         return value * factorial(value - 1)
+
+def is_prime(value):
```

Comment:
This function is well written. Docstrings are missing, but these are not necessary for now.